src/idom/web/module.py

```diff
@@ -83,6 +88,7 @@ def module_from_template(
     resolve_exports: bool = IDOM_DEBUG_MODE.current,
     resolve_exports_depth: int = 5,
     unmount_before_update: bool = False,
+    replace_existing: bool = False,
```

Comment:
A better default for this is `True`. Most developers would expect this behavior.

Reply:
I don't want someone to silently and accidentally replace a module they didn't mean to. Because this directory is shared globally, libraries need to cooperate with respect to what they name their bundled module files. Ideally every module should correspond to a package published to NPM, but we can't necessarily rely on that.

Reply:
Perhaps a global setting that can configure this value then?

At least within Django-IDOM, it'd be nice to have some way to set this as default.

Reply:
> I'm thinking to put the files directly into the Django-IDOM app directory

So copying files from `IDOM_WEB_MODULES_DIR` into the app directory, or the other way around?

Reply:
My thought process was to either set `IDOM_WEB_MODULES_DIR` to `<django_idom>/data/web_modules/` by default and letting the user further configure this directory through `settings.py`.

The alternatives are symlinking or copying them over.

At some point, we're going to be putting the web module files directly into `django_idom/static/` whenever I get around to [this ticket.](https://github.com/idom-team/django-idom/issues/4)

Reply:
Perhaps I could log a warning instead of raising?

Reply:
Agreed, it's not a critical failure so it shouldn't be an exception.